### DISCUSSION TITLE
Different UI for Mobile

### DISCUSSION THREAD

**Discussion by TheRobo-13** [unoplatform/uno#8009] [Q&A]
how do i make a project have different ui for mobile?

**Comment by pkar70** [comment]
Can you be more specific?
You can assume that iOS and Android are mobile, Windows and macOS are desktop. So, you can use conditional XAML (as  e.g. <win:TextBox)

**Comment by workgroupengineering** [comment]
See [there](https://platform.uno/docs/articles/platform-specific-xaml.html?q=condition).
